packages/table-renamer/index.ts

```diff
@@ -0,0 +1,162 @@
+import { GetParameterCommand, SSMClient } from "@aws-sdk/client-ssm";
+import { Kysely, sql } from "kysely";
+import * as logger from "lambda-log";
+
+import { Database } from "@create-disruptions-data/shared-ts/db/types";
+import { getDbClient } from "@create-disruptions-data/shared-ts/utils/db";
+import {
+    disableTableRenamerParamName,
+    putTableRenamerDisableParameter,
+} from "@create-disruptions-data/shared-ts/utils/ssm";
+
+const ssm = new SSMClient({ region: "eu-west-2" });
+
+export const main = async () => {
+    const { STAGE: stage } = process.env;
+    try {
+        logger.info("Table Renamer starting... ");
+
+        const dbClient = getDbClient();
+
+        let disableRenamer: string | undefined = "true";
+
+        if (!stage) {
+            throw new Error("Stage env not found");
+        }
+
+        try {
+            const input = {
+                Name: `${disableTableRenamerParamName}-${stage}`,
+            };
+            const command = new GetParameterCommand(input);
+            const ssmOutput = await ssm.send(command);
+            disableRenamer = ssmOutput.Parameter?.Value;
+        } catch (error) {
+            if (error instanceof Error) {
+                throw new Error(`Failed to get parameter from ssm: ${error.stack || ""}`);
+            }
+
+            throw error;
+        }
+
+        if (disableRenamer === "false") {
+            await checkReferenceDataImportHasCompleted("operator_lines", dbClient);
+            await checkReferenceDataImportHasCompleted("operator_public_data", dbClient);
+            await checkReferenceDataImportHasCompleted("operators", dbClient);
+            await checkReferenceDataImportHasCompleted("stops", dbClient);
+            await checkReferenceDataImportHasCompleted("services", dbClient);
+            await checkReferenceDataImportHasCompleted("localities", dbClient);
+            await checkReferenceDataImportHasCompleted("nptg_admin_areas", dbClient);
+
+            await deleteAndRenameTables(dbClient);
+        } else {
+            await putTableRenamerDisableParameter(stage, "false", logger);
```

Comment:
We should be able to get rid of this now since the error handling will be performed by the step function instead